integration-tests/src/test/java/oracle/weblogic/kubernetes/ItRetryImprovementMisc.java

```diff
@@ -0,0 +1,471 @@
+    assertTrue(patchClusterCustomResource(clusterName, domainNamespace,
+        patch, V1Patch.PATCH_FORMAT_JSON_PATCH), "Failed to patch cluster");
+
+    // check the domain Complete goes to false
+    checkDomainStatusConditionTypeHasExpectedStatus(shortRetryPolicy, domainUid, domainNamespace,
+        DOMAIN_STATUS_CONDITION_COMPLETED_TYPE, "False", DOMAIN_VERSION);
+
+    // Enable this check when OWLS-104622 is fixed
+    // check the cluster Complete should go false
+    //checkDomainStatusClusterConditionTypeHasExpectedStatus(shortRetryPolicy, domainUid, domainNamespace,
+    //    clusterName, DOMAIN_STATUS_CONDITION_COMPLETED_TYPE, "False", DOMAIN_VERSION);
+
+    // verify Cluster Available should go to false and stay false
+    checkDomainStatusClusterConditionTypeHasExpectedStatus(withQuickRetryPolicy, domainUid, domainNamespace,
+        clusterName, DOMAIN_STATUS_CONDITION_AVAILABLE_TYPE, "False", DOMAIN_VERSION);
+
+    // verify Domain available should stay true throughout
+    checkDomainStatusConditionTypeHasExpectedStatus(withQuickRetryPolicy, domainUid, domainNamespace,
+        DOMAIN_STATUS_CONDITION_AVAILABLE_TYPE, "True", DOMAIN_VERSION);
+
+    // check the cluster server pod is deleted
+    checkPodDoesNotExist(managedServerPrefix + 1, domainUid, domainNamespace);
+
+    // check the cluster Complete should go true
+    checkDomainStatusClusterConditionTypeHasExpectedStatus(withQuickRetryPolicy, domainUid, domainNamespace,
+        clusterName, DOMAIN_STATUS_CONDITION_COMPLETED_TYPE, "True", DOMAIN_VERSION);
+    // verify Cluster Available should stay false
+    checkDomainStatusClusterConditionTypeHasExpectedStatus(withQuickRetryPolicy, domainUid, domainNamespace,
+        clusterName, DOMAIN_STATUS_CONDITION_AVAILABLE_TYPE, "False", DOMAIN_VERSION);
+
+    // verify Domain available should stay true throughout
+    checkDomainStatusConditionTypeHasExpectedStatus(withQuickRetryPolicy, domainUid, domainNamespace,
+        DOMAIN_STATUS_CONDITION_AVAILABLE_TYPE, "True", DOMAIN_VERSION);
+    // check the domain Complete back to True when the cluster shutdown completes
+    checkDomainStatusConditionTypeHasExpectedStatus(withQuickRetryPolicy, domainUid, domainNamespace,
+        DOMAIN_STATUS_CONDITION_COMPLETED_TYPE, "True", DOMAIN_VERSION);
+  }
+
+  /**
+   * Try set cluster replicas=2. The cluster Complete should immediately go false, and cluster Available should go true
+   * the moment one of the cluster's members becomes ready, and cluster Complete should become true once both servers
+   * are ready. Domain available and complete should both flip to false, domain available should become true when
+   * cluster available becomes true, and domain complete should only come back to true when the cluster is fully up
+   * and ready.
+   */
+  @Order(6)
+  @Test
+  @DisplayName("Set cluster replicas to 2, verify the domain and cluster status Complete and Available condition")
```

Comment:
@DisplayName("Set cluster replicas to 2, verify the domain and cluster status Complete and Available condition") -->
@DisplayName("Verify the domain and cluster status Complete and Available condition after scaling up the cluster")